### PR TITLE
Added tabs in vulnerability view

### PR DESCRIPTION
**Pull request by Cheshiriks**
- added tabs in vulnerability view
- remove project tables in FossGraphView

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt [hunk 1]
```diff
@@ -264,7 +109,7 @@ val fossGraph: FC<FossGraphViewProps> = FC { props ->
 
             if (isSuperAdmin || isOwner) {
                 buttonBuilder(label = "Delete", style = "danger", classes = "mr-2") {
-                    setDeleteProject(null)
+                    // setDeleteProject(null)
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt:112]: Why so?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/VulnerabilityInfoTab.kt [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+    @Suppress(
+        "TYPE_ALIAS",
+        "MAGIC_NUMBER",
+    )
+    val openSourceProjectTable: FC<TableProps<VulnerabilityProjectDto>> = tableComponent(
+        columns = {
+            columns {
+                column(id = "name", header = "Name", { name }) { cellContext ->
+                    Fragment.create {
+                        ReactHTML.td {
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/VulnerabilityInfoTab.kt:86]:
```suggestion
                        td {
```

and so on

**Review by sanyavertolet — APPROVED** [review]
Screenshot might be nice

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt:112]: Forgot to delete.
It is strange that the Diktat did not notice this.

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/VulnerabilityInfoTab.kt (hunk 2 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/VulnerabilityInfoTab.kt]: Fixed

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt (hunk 1 above)
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt]: Sometimes diktat skips those commented code fragments

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/VulnerabilityInfoTab.kt [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+import react.*
+import react.dom.html.ReactHTML.div
+import react.router.dom.Link
+import web.cssom.ClassName
+
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import react.dom.html.ReactHTML.td
+
+val renderVulnerabilityInfo: FC<VulnerabilityInfoTabProps> = FC { props ->
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/VulnerabilityInfoTab.kt:26]: pls change the name in future PR. Component name should not start with render, as it is function prefix

Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/VulnerabilityInfoTab.kt]: Renamed

**Pull request merged**